tests/test_knoema_data_time.py

```diff
@@ -0,0 +1,60 @@
+"""This is test module for knoema client with test credentials"""
+
+import unittest
+import knoema
+import os
+
+class TestKnoemaClient(unittest.TestCase):
+    """This is class with knoema client unit tests with test credentials"""
+
+    base_host = 'knoema.com'
+
+    def setUp(self):
+        apicfg = knoema.ApiConfig()
+        apicfg.host = os.environ['BASE_HOST'] if 'BASE_HOST' in os.environ else self.base_host
+        apicfg.app_id = os.environ['KNOEMA_APP_ID'] if 'KNOEMA_APP_ID' in os.environ else ''
+        apicfg.app_secret = os.environ['KNOEMA_APP_SECRET'] if 'KNOEMA_APP_SECRET' in os.environ else ''
+
+    def test_getdata_by_TransformationDataReader_timemembers(self):
+        """The method is testing getting single series by dimension member ids"""
+
+        data_frame = knoema.get('IMFWEO2021Apr', Country='614', Subject='BCA', timemembers='1980;2002;2023')
```

Comment:
Why timemembers is string? Could we use array and internally join items to string?

Reply:
the same as for all parameters: several dimension members, multiple frequencies